### PR TITLE
[V1.4] Fix Undo, Redo

### PR DESCRIPTION
**Pull request by EdmondOng**

### CODE REVIEW

File: src/main/java/nasa/ui/activity/DeadlineCard.java [hunk 1]
```diff
@@ -53,6 +55,20 @@ public DeadlineCard(Deadline activity) {
...
         note.setText(activity.getNote().toString());
         status.setText(activity.getStatus().toString());
+        priority.setText(activity.getPriority().toString());
+        dateToRepeat.setText("-");
+        if (activity.getSchedule().getType() != 0) {
+            dateToRepeat.setText("Repeat: " + activity.getScheduleDate().toString());
+        }
+        int urgent = activity.getDifferenceInDate();
+        if (urgent > 5) {
+            circle.setFill(Color.GREEN);
```
Comment (don-tay) [inline src/main/java/nasa/ui/activity/DeadlineCard.java:65]: The team prefers for the UI to do away with the circle indicator as it is not as aesthetically pleasing. Is this arrangement okay for you?

File: src/main/java/nasa/ui/activity/QuotePanel.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import nasa.ui.UiPart;
+
+/**
+ * UI component for quote panel.
+ */
+public class QuotePanel extends UiPart<Region> {
+
+    private static final String FXML = "QuotePanel.fxml";
+    private int time = 5000;
+
```
Comment (don-tay) [inline src/main/java/nasa/ui/activity/QuotePanel.java:20]: Please screenshot or video demo how this quote panel look like. Thank you.

Comment (EdmondOng) [reply src/main/java/nasa/ui/activity/QuotePanel.java:20]: image

On start up, there will be a quote message that will appear for 5s.

File: src/main/java/nasa/ui/activity/DeadlineCard.java (hunk 1 above)
Comment (EdmondOng) [reply src/main/java/nasa/ui/activity/DeadlineCard.java:65]: image
If the team dont like than is ok.

**Pull request closed without merging**